var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -43,16 +43,12 @@ class Metis(Package):
     version('5.0.2', 'acb521a4e8c2e6dd559a7f9abd0468c5')
     version('4.0.3', 'd3848b454532ef18dc83e4fb160d1e10')
 
-    variant('shared', default=True,
-            description='Enables the build of shared libraries')
-    variant('debug', default=False,
-            description='Builds the library in debug mode')
-    variant('gdb', default=False, description='Enables gdb support')
+    variant('shared', default=True, description='Enables the build of shared libraries.')
```

Comment:
I thought that @tgamblin just used a script to ensure that each file followed the style guidelines and that the variant/version directives were accidentally updated as a result (lines with these directives were exempt from the 80 character line rule based on a decision we made in PR #961).  Has there been a change in the style guidelines regarding version/variant lines since #961?  If so, I'll revise the `share/spack/qa/run-flake8-tests` file and revert these style changes.